### ISSUE TITLE
EL onboarding materials updates 

### ISSUE THREAD

**Issue by samumantha** [coderefinery/manuals#239]
For September 2023 workshop we will have two EL onboarding sessions, one for online and one for in-person team leads.

Task is to make sure that the manuals are up to date enough for providing this. Some info is relevant to both, other only to one group. 

- [ ] separate online / in-person EL materials somehow
- [ ] for online, differentiate between public zoom and own online team
- [ ] as the learner survey is old anyway, show the most important aspects (showcasing variety of learners, their background and infrastructure) of it on onboarding page (less switching tabs)
- [ ] improve flow of materials
- [ ] ...more to come while going through materials, please add if you have anything :)

**Comment by samumantha** [comment]
Separation: Maybe still better to leave it all in one page? 
Then it is just up to the EL onboarding leader to leave out the part that is not interesting for the current group (to stay in time)

**Comment by bast** [comment]
These are great points!
- I agree that one page is probably better, easier to maintain, less risk of diverging and one getting forgotten
- Less is more
- Generally let us try to reduce the amount of text: easier for the reader but also easier for the presenter

**Comment by bast** [comment]
I believe it is this page which we will present and go through and which we are improving now? https://coderefinery.github.io/manuals/team-leaders/

**Comment by bast** [comment]
If yes it would be good if we manage to reduce text by 50% so that there is less risk that we forget important points and to free up more time for discussions and Q&A during on-boarding.

**Comment by samumantha** [comment]
I agree with that, but do not want onto anybodys toes who spent time writing all these valuable info. Do the manuals support spoiler or tabs? Or should we have a more textual page for manuals and a more limited page for the onboarding?

**Comment by bast** [comment]
I will link to https://coderefinery.github.io/manuals/team-leaders/ from schedule.

**Comment by samumantha** [comment]
Which should then be the short version used for onboarding?

**Comment by bast** [comment]
I find that reducing text does not undo or invalidate the work of those who added text.

But if we want to keep text, I would not split into two pages but rather that we use elements where somebody can unroll text if they want to read more. I don't think splitting information over more than one page will be maintainable. One of them will get outdated.

**Comment by bast** [comment]
If we don't have webpage elements to help with that, we can have short info on top of page which we go through in on-boarding and which somebody can read if they have 5 minutes, and below we can have longer text for those who want to read more.
